src/status_im/native_module/core.cljs

```diff
@@ -35,6 +35,10 @@
   (clear-web-data)
   (.saveAccountAndLogin (status) multiaccount-data hashed-password config accounts-data))
 
+(defn save-account-and-login-with-keycard
```

Comment:
could you add a docstring that explains what this does as well as a description of the arguments, particularly chat-key

Reply:
don't forget to document these other devs are going to have a hard time figuring out what these arguments are and why they are necessary